httpbakery/client_test.go

```diff
@@ -34,6 +33,8 @@ type ClientSuite struct {
 
 var _ = gc.Suite(&ClientSuite{})
 
+var macaroonCurrentVersion = bakery.MacaroonVersion(bakery.LatestVersion)
```

Comment:
Is this safe to do always? If it is then please add a comment explaining why, otherwise perhaps use a bakery.MacaroonVersion constant.